src/app/timer/timer.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit } from '@angular/core';
+import { interval, subscribeOn, Subscription, timer, fromEvent, from } from 'rxjs';
+
+
+@Component({
+  selector: 'app-timer',
+  templateUrl: './timer.component.html',
+  styleUrls: ['./timer.component.css'],
+})
+export class TimerComponent implements OnInit {
+
+  //Ненужная переменная, просто для гита
+  bgit = 'Lorem'
+  
+  
+  constructor() {
+
+  }
+
+  hours = 0;
+  minuts = 0;
+  seconds = 0;
+
+  text:string = "pause"
+
+  date = new Date(this.hours, this.minuts, this.seconds);  
+
+  myTimer = timer(0, 1000);
+
+  time!: Subscription;
+
+  isWork = false;
+
+  condition = 'Start';
+
+  startTimer() {
+    if(this.isWork === false) {
+      this.time = this.myTimer.subscribe(() => {
+      this.seconds++
+      this.date.setHours(this.hours, this.minuts, this.seconds);
+      this.isWork = true
+      this.condition = 'Res'
+    });
+    } else {
+      this.seconds = this.seconds;
+      this.date.setHours(this.hours = 0, this.minuts = 0, this.seconds = 0);
+      this.condition = 'Start';
+      this.isWork = false;
+      this.time.unsubscribe()
+    }
```

Comment:
this part of function is actually doing reset functionality, so method name is wrong
let's try to use separate functions for these actions (start, reset)